modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/index/ComplexSecondaryKeyUnwrapSelfTest.java

```diff
@@ -55,15 +52,15 @@ public void testSecondaryIndexWithIntersectColumnsComplexPk() {
 
         executeSql("CREATE INDEX ON " + tblName + "(id, name, city)");
 
-        checkUsingIndexes(tblName, "'1'");
+        checkUsingIndexes(tblName, "'1'", 2, "Query should be splitted");
     }
 
     /**
      * Test using secondary index with simple PK.
      */
     @Test
     public void testSecondaryIndexSimplePk() {
-        HashMap<String, String> types = new HashMap() {
+        Map<String, String> types = new LinkedHashMap<String, String>() {
```

Comment:
What is a purpose of this change?

Reply:
The order of types from the declaration is cosily for debugging. Just for this. But I can revert it back.

Reply:
OK, let's use LinkedHashMap then.